etl/src/main/java/org/pih/hivmigration/etl/sql/Migrator.java

```diff
@@ -171,6 +171,7 @@ public void main() {
                 migrate(new AccompagnateurMigrator(), -1);
                 migrate(new SocioEconomicAssistanceMigrator(), -1);
                 migrate(new VisitMigrator(), limit);
+                migrate(new UnknownEncounterLocationsMigrator(), -1);
                 if (devFeatures) {
```

Comment:
I think we've reached the point that we are no longer really using or can rely on the revert functionality, so not sure if it would be worth adding here... undoing the location changes should be pretty straightforward since we are logging the data migrations, but not sure about the downstream stuff, or if we are ever going to use this functionality going forward.